### PR TITLE
Release v0.1.0

### PR DESCRIPTION
**Pull request by TraGicCode**
@bastelfreak, Not really sure how the changelog should be for the initial release.  Let me know if i need to make any changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ The stackify module requires the following:
 * Puppet Agent 4.0 or later.
```
Comment (bastelfreak) [inline README.md]: can you bump this to 4.7.1? This is the lowest version that Vox Pupuli supports.

File: metadata.json [hunk 2]
```diff
@@ -1,14 +1,30 @@
...
   "summary": "Installs stackify APM agent",
   "license": "Apache-2.0",
-  "source": "https://github.com/tragiccode/tragiccode-stackify",
-  "project_page": "https://github.com/tragiccode/tragiccode-stackify",
-  "issues_url": "https://github.com/tragiccode/tragiccode-stackify/issues",
+  "source": "https://github.com/voxpupuli/puppet-stackify",
+  "project_page": "https://github.com/voxpupuli/puppet-stackify",
+  "issues_url": "https://github.com/voxpupuli/puppet-stackify/issues",
   "dependencies": [
     {"name":"puppetlabs-stdlib","version_requirement":">= 4.0.0 < 5.0.0"}
```
Comment (bastelfreak) [inline metadata.json]: please bump stdlib to 4.13.1

File: metadata.json [hunk 3]
```diff
@@ -1,14 +1,30 @@
...
+        "2012",
+        "2012R2",
+        "2016"
+      ]
+    }
+  ],
+  "requirements": [
+    {
+      "name": "puppet",
+      "version_requirement": ">= 4.0.0 < 6.0.0"
```
Comment (bastelfreak) [inline metadata.json]: please bump puppet to 4.7.1

File: metadata.json [hunk 4]
```diff
@@ -1,14 +1,30 @@
...
+    }
+  ],
   "data_provider": null
```
Comment (bastelfreak) [inline metadata.json]: I am not sure if data_provider is still valid? It may got deprecated.

File: README.md (hunk 1 above)
Comment (TraGicCode) [reply README.md]: Done.

File: metadata.json (hunk 3 above)
Comment (TraGicCode) [reply metadata.json]: Done.

File: metadata.json (hunk 4 above)
Comment (TraGicCode) [reply metadata.json]: Removed and Done.

**Comment by TraGicCode** [comment]
@bastelfreak Everything should be good to go now.  Resolved all the issues you had raised.

**Pull request merged**